BUG Fix: The showSlides() had a BUG as it was nested inside another Function so it was not Accessible Globally.

pratik-k-ghosh opened the pull request:
…on and solved the BUG: showSlides()

Commit:
Removed Unwanted Functions, Increased Readability, better Documentation and solved the BUG: showSlides()

pratik-k-ghosh commented:
BUG Fixed: the showSlides() was nested inside currentSlide() so showSlides() was not Accessible Globally.

Commit:
Added Favicon

Devesh1509 commented:
@PratikKrGhosh, Great.. if you have solved the bug... Please share the snapshots after your changes.

Commit:
Bug Fix: fixed img src Bug

Devesh1509 commented:
@PratikKrGhosh Please share the snapshots after your changes.

Commit:
BugFix: Fixed favicon Bug

pratik-k-ghosh commented:
@Devesh1509 Sure Here is the Snapshot.

https://github.com/Devesh1509/JavaScript-Slider/assets/144181107/ede1236a-ab21-48d4-8dd5-4d16cd0d7ace

pratik-k-ghosh commented:
Finally I have
1) added Favicon to Your Website
2) Fixed the showSlides() Bug
3) Increased Program Readability
4) done Documentation by adding Comments in JS file.
5) Removed Unwanted lines of code by Commenting them out.
